chore(server): update typeorm

michelheusschen opened the pull request:
Updates TypeORM from version `0.3.11` to `0.3.12`, which includes a fix for redundant unique constraints. As a result, a migration file is necessary

michelheusschen committed:
chore(server): update typeorm

zackpollard commented:
Did TypeORM generate these migrations itself? Can you confirm there are other constraints that still fulfill this criteria as I do believe we still want this to be unique, or is this handled directly by the column definition now?

michelheusschen commented:
TypeORM generated these, but I manually removed some index recreations. `assetId` is the primary key of both the `exif` and `smart_info` tables, there was only an additional unique constraint generated by TypeORM which is now fixed and removed by these migrations.
